src/agents/AgentCoordinator.ts

```diff
@@ -17,12 +18,14 @@ export class AgentCoordinator {
     @inject(UniswapPriceScannerAgent) UniswapPriceScannerAgent: UniswapPriceScannerAgent,
     @inject(UniswapVerificationAgent) UniswapVerificationAgent: UniswapVerificationAgent,
     @inject(FeedAgent) FeedAgent: FeedAgent,
+    @inject(BlockMintingAgent) BlockMintingAgent: BlockMintingAgent,
   ) {
     this.agents = {
       UniswapPoolScannerAgent,
       UniswapPriceScannerAgent,
       UniswapVerificationAgent,
       FeedAgent,
+      BlockMintingAgent,
```

Comment:
I think we should use bullMQ here. I won't change it now because this PR is too old and will be merged into another feature branch that we can work on.